CHANGELOG.md

```diff
@@ -1,3 +1,36 @@
+# v0.12.0
+
+## Added
+
+- [[#187](https://github.com/rust-vmm/kvm-ioctls/pull/187)] Support for 
+  `KVM_SET_IDENTITY_MAP_ADDR`
+- Derive Debug for all exported structs and enums
+- [[#189](https://github.com/rust-vmm/kvm-ioctls/pull/189)] Expose `KVM_SET_` and 
+  `KVM_HAS_DEVICE_ATTR` for vcpu
+- [[#191](https://github.com/rust-vmm/kvm-ioctls/pull/191)] Add `KVM_TRANSLATE` and
+  `translate_gva` support
+- [[#190](https://github.com/rust-vmm/kvm-ioctls/pull/190)] Enable usage of `sync_regs`
+- [[#198](https://github.com/rust-vmm/kvm-ioctls/pull/198)] Expose details from 
+  `KVM_EXIT_FAIL_ENTRY`
+- [[#199](https://github.com/rust-vmm/kvm-ioctls/pull/199)] Add `register_irgfq_with_resample`
```

Comment:
Typo in function name: s/register_irgfq_with_resample/register_irqfd_with_resample

Reply:
Fixed